e2e_tests.py

```diff
@@ -4,19 +4,24 @@
 from utils import type_check, timems
 import urllib.parse
 from config import config
+import sys
+import threading
 
 host = 'http://localhost:' + str (config ['port']) + '/'
+hosts = {'alpha': 'https://hedy-alpha.herokuapp.com/', 'test': 'https://hedy-alpha.herokuapp.com/'}
```

Comment:
Typo in the `test` field?

Reply:
Why is the full hostname not just `sys.argv[1]` or something?

Reply:
Fixed the typo, thanks for the catch. The full hostname would be cumbersome to type. I'd rather have a friendly and meaningful alias for the environment, such as `alpha` or `test`.